source/javascripts/site.js

```diff
@@ -2,7 +2,10 @@
   // Resize all graphs that need it
   var resizeGraphs = function() {
     $('.plotly-graph-needs-resize').each(function() {
-      Plotly.relayout(this.id, {});
+      let id = this.id;
+      requestAnimationFrame(function() {
```

Comment:
I'm not too up on `requestAnimationFrame `, is this widely available enough to assume existence or should a polyfill be added to be safe?

Reply:
I suspect it's widely available enough to be depended on without polyfill https://caniuse.com/requestanimationframe.

If you'd like to support IE 6-9 and Opera Mini support, happy to add a polyfill though.

Reply:
:+1: Thanks for the link, that looks good enough I think, not that a polyfill would hurt.